packages/core/src/QuickLInks/index.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import clsx from "clsx";
+
+import { Link, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    [theme.breakpoints.up("md")]: {
+      paddingBottom: "0.5rem",
+    },
+  },
+  list: {
+    listStyle: "none",
+    padding: 0,
+    paddingTop: "1rem",
+    "& > li": {
+      marginTop: "2.3125rem",
+    },
+    [theme.breakpoints.up("md")]: {
+      "& > li": {
+        marginTop: "1.5rem",
+      },
+    },
+  },
+  link: {
+    ...theme.typography.caption,
+    textDecoration: "none",
+  },
+  title: {
+    lineHeight: "inherit",
+  },
+}));
+
+function QuickLinks({ options, linkComponent, links, title, ...props }) {
+  const classes = useStyles(props);
+  const LinkComponent = linkComponent || Link;
+
+  return (
+    <div className={classes.root}>
+      <Typography variant={options.title.variant} className={classes.title}>
+        {title}
+      </Typography>
+      <Typography
+        variant={options.link.variant}
+        className={clsx([classes.text, classes.list])}
+        component="ul"
+      >
+        {links.map(({ label, ...others }) => (
+          <li key={label}>
+            <LinkComponent
+              {...others}
+              color={options.link.color}
+              variant={options.link.variant}
+              underline="none"
+              className={classes.link}
+            >
+              {label}
+            </LinkComponent>
+          </li>
+        ))}
+      </Typography>
+    </div>
+  );
+}
+
+QuickLinks.propTypes = {
+  linkComponent: PropTypes.elementType,
+  links: PropTypes.arrayOf(PropTypes.shape({}).isRequired).isRequired,
+  options: PropTypes.shape({
```

Comment:
I think we should drop the options and let people set them using CSS. Similar to what Material UI does.
`classes={{root:some classes, title:some classes}}`

Reply:
@KobbyMmo If someone is not using MaterialUI in their project do we dictate them to use Material's styling library to pass in the styles?

Reply:
I think if you are not using Material UI it still works because you will pass something like
`classes={{root:"root-a", title:"title-a"}}`

and in your own component, you will have your style definition like
 ```
root-a{
margin:0
border:0
}
```

Reply:
@KobbyMmo Okay, let me change that

Reply:
@KobbyMmo Um, I think we should leave `options` as is because it just set the `default` when we have no CSS styling on Quicklinks. What I'll do is accommodate for the CSS styling the way Material ui does as you mentioned above